datadog/table_datadog_service_level_objective.go

```diff
@@ -0,0 +1,112 @@
+package datadog
+
+import (
+	"context"
+
+	datadog "github.com/DataDog/datadog-api-client-go/api/v1/datadog"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+func tableDatadogServiceLevelObjective(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "datadog_service_level_objective",
+		Description: "An SLO(Service Level Objective) provides a target percentage of a specific metric over a certain period of time.",
+		Get: &plugin.GetConfig{
+			Hydrate:    getSLO,
+			KeyColumns: plugin.SingleColumn("id"),
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listSLOs,
+			KeyColumns: plugin.KeyColumnSlice{
+				{Name: "id", Require: plugin.Optional},
```

Comment:
Is it necessary to add `Id` as an optional qual? 
If we will run the query `select * from datadog_service_level_objective where id = 'xxxxx';` I believe the get API will be called.